### PR TITLE
Added FIAAS Python client library

### PR DESCRIPTION
**Pull request by mortenlj**
I have added the FIAAS Python client library. The library is still in development, and not complete. It has a different approach than most other libraries, which some developers might find more suited to their tastes.

### CODE REVIEW

File: content/en/docs/reference/using-api/client-libraries.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ their authors, not the Kubernetes team.
 | Python               | [github.com/eldarion-gondor/pykube](https://github.com/eldarion-gondor/pykube) |
 | Python               | [github.com/mnubo/kubernetes-py](https://github.com/mnubo/kubernetes-py) |
 | Python               | [github.com/tomplus/kubernetes_asyncio](https://github.com/tomplus/kubernetes_asyncio) |
+| Python               | [github.com/fiaas/k8s](https://github.com/fiaas/k8s) |
```
Comment (sftim) [inline content/en/docs/reference/using-api/client-libraries.md]: Please add this into the existing sort order (fiaas sorts after eldarion-gondor)

Comment (mortenlj) [reply content/en/docs/reference/using-api/client-libraries.md]: Ooops, my bad. I didn't realise they were sorted. Will fix asap.

Comment (mortenlj) [reply content/en/docs/reference/using-api/client-libraries.md]: @sftim : Fixed (after a git misfire)

**Comment by mortenlj** [comment]
Hi. What can I do to get this merged?

**Pull request merged**